### PR TITLE
game like cv

### PR DESCRIPTION
**Pull request by yonbav**
We added alot of assets. 
We are still working on it and plan to work on it, but all unused assets will be removed when we are done.

### CODE REVIEW

**Review by tsachis — COMMENTED** [review]
I'll merge it manually as there is some conflicts and you touched some file outside of the skin folder

File: src/index.scss [hunk 1]
```diff
@@ -1,14 +1,14 @@
 body {
```
Comment (dsternlicht) [inline src/index.scss:1]: I don't like that all of these changes are on the main `index.scss` file. They might affect all other skins. Can this be scoped to your skin only?

File: src/models.ts [hunk 2]
```diff
@@ -316,7 +316,7 @@ export interface Work {
   /**
```
Comment (dsternlicht) [inline src/models.ts:316]: @tsachis Can you please review?

**Review by dsternlicht — CHANGES_REQUESTED** [review]

**Comment by tsachis** [comment]
already merged it manually

**Pull request closed without merging**